dev/com.ibm.ws.jaxrs.2.1.sse_fat/test-applications/BasicSseApp/src/jaxrs21sse/basic/BasicSseTestServlet.java

```diff
@@ -396,9 +398,14 @@ public void testErrorSse(HttpServletRequest req, HttpServletResponse resp) throw
 
                                 @Override
                                 public void accept(InboundSseEvent t) {
+                                    try {
```

Comment:
Is this try/catch part of the debug that you are planning to remove?  If not, can you indent the code in the try block?

Reply:
Done